browser/src/components/pages/Home.vue

```diff
@@ -52,19 +55,17 @@
 </template>
 
 <script lang="ts">
-import Disco from '@/assets/svg/Disco.vue'
 import ButtonCard from '@/components/containers/ButtonCard.vue'
 import { defineComponent } from 'vue'
 
 export default defineComponent({
   name: 'Home',
   components: {
-    ButtonCard,
-    Disco
+    ButtonCard
   },
   data () {
     return {
-      discoWelcome: require('../../assets/public/discoWelcome500.gif')
+      discoWelcome: require('../../assets/public/disco_middle.gif')
```

Comment:
it might be a good idea to make a component out of GIFs, similar to SVGs (`src/assets/svg`)